### ISSUE TITLE
Too few transactions are included when Executing Batch 

### ISSUE THREAD

**Issue by bingclo** [5afe/safe-support#258]
### Issue Category

Transaction execution issue

### What happened?

User wanted to Execute multiple USDT transfers that are already in Queue (all fully signed) but only 19 transactions are included in a batch when clicking on Execute Batch button. I've impersonated his owner wallet and seeing the same issue. 
![image](https://github.com/5afe/safe-support/assets/54754746/12dd2ea6-05f8-47af-b5a0-9b805f84b480)

When simulating the batch transaction it shows that the gas limit used for the 19 transactions is already at 30M gas. 
https://dashboard.tenderly.co/public/safe/safe-apps/simulator/4e639d7c-ee58-498e-8f7f-becd2a7a0e7a

User expected to execute more than the 19 transactions which is currently up to nonce 54 only. 

Is this normal? How much extra gas/overhead is used when estimating gas limit for these transactions?

### Date and time this happened or you first noticed this issue

0/08/2023 00:55:06

### Network

Ethereum Mainnet

### Safe Address

https://app.safe.global/transactions/queue?safe=eth:0xe1D29D8970d8333d22f59775D22aF22200D6BA13

### Token Address

0xdac17f958d2ee523a2206206994597c13d831ec7

### Token Type

ERC20

### Transactions

```shell
N/A
```

**Comment by katspaugh** [comment]
@usame-algan there's a limit of 20 AFAIR?

**Comment by usame-algan** [comment]
> @usame-algan there's a limit of 20 AFAIR?

Yes, 20 is currently a UI limitation we set because of the pagination.

**Comment by johannesmoormann** [comment]
To be closed. We do not support the batching of more transactions for the moment.
